controllers/seedimage_controller.go

```diff
@@ -43,14 +43,21 @@ import (
 	"github.com/rancher/elemental-operator/pkg/util"
 )
 
-const seedImgVolSize = 4 * 1024 * 1024 * 1024 // 4 GiB
+const seedImgVolSize = 6 * 1024 * 1024 * 1024 // 6 GiB
```

Comment:
I guess not for now but this should be configurable IMHO, as long as we allow to add custom builders I think this starts to be a hard requirement having a chance to set it to different values.

Reply:
Yep I was thinking of setting it as a value in the Spec instead.. Will actually try to add that as well!